nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/test/java/org/apache/nifi/web/api/TestFlowResource.java

```diff
@@ -502,6 +506,98 @@ public void testActivateControllerServicesStateEnabled() {
         assertEquals(expectedServicesIds, serviceIds);
     }
 
+    @Test
+    public void testClearBulletinsIncludesControllerServices() {
```

Comment:
This method does not call `clearBulletins`, so it does not seem to be verifying the changes.